pkg/providers/instance/instance.go

```diff
@@ -259,9 +264,6 @@ func getTags(ctx context.Context, nodeClass *v1beta1.EC2NodeClass, nodeClaim *co
 			v1alpha5.MachineManagedByAnnotationKey:                                         settings.FromContext(ctx).ClusterName,
 		}
 	} else {
-		overridableTags = map[string]string{
-			"Name": fmt.Sprintf("%s/%s", corev1beta1.NodePoolLabelKey, nodeClaim.Labels[corev1beta1.NodePoolLabelKey]),
-		}
```

Comment:
Was this intentional? Will this break users?

Reply:
Yes it was intentional. Since this only applies to `NodePool`s it won't break users who are still on v1alpha5. The main intention of this PR is to change the value of the `Name` tag to reflect the name of the node rather than the nodepool so that would be a breaking change as part of v1beta1.